armada_command/command_stop.py

```diff
@@ -38,7 +38,9 @@ def command_stop(args):
                 'Provide more specific container_id or microservice name or use -a/--all flag.'.format(**locals()))
         print('Stopping {instances_count} services {microservice_name}...'.format(**locals()))
     else:
-        print('Stopping service {microservice_name}...'.format(**locals()))
+        microservice_name = instances[0]['ServiceName']
```

Comment:
It shadows microservice_name in line 27 (https://github.com/armadaplatform/armada/pull/3/files#diff-8f1617b003e0be99667803b8fed70dd4L27). I would suggest changing that one to something like microservice_name_or_id.